src/hooks/useDrag.tsx

```diff
@@ -494,12 +600,14 @@ export const onDelete = (
       const sections = deleteElement(frontMatter.sections, indexToDelete)
       const newErrorSections = deleteElement(errors.sections, indexToDelete)
       const newDisplaySections = deleteElement(displaySections, indexToDelete)
+      const newTextcardErrors = deleteElement(errors.textcards, indexToDelete)
 
       return updateEditorSection(
         homepageState,
         newDisplaySections,
         sections,
-        newErrorSections
+        newErrorSections,
+        newTextcardErrors
       )
```

Comment:
if my understanding here is correct, `homepageState.errors.textcards[i]` would already have been deleted, no? so we don't need to pass in this argument

Reply:
Ah nope, `errors.textcards` is an nested array - each textcard error is identified by parent index and subitem index, and the only time the first level of `errors.textcards` is modified is when a section is created/deleted/moved